website/docs/r/appfabric_app_authorization.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+subcategory: "AppFabric"
+layout: "aws"
+page_title: "AWS: aws_appfabric_app_authorization"
+description: |-
+  Terraform resource for managing an AWS AppFabric App Authorization.
+---
+
+# Resource: aws_appfabric_app_authorization
+
+Terraform resource for managing an AWS AppFabric App Authorization.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+resource "aws_appfabric_app_authorization" "example" {
+  app                   = "TERRAFORMCLOUD"
+  app_bundle_identifier = aws_appfabric_app_bundle.arn
```

Comment:
include a resource for app_bundle.arn and the formatting is incorrect. You might want to consider aws_appfabric_app_bundle.test.arn

Reply:
Yes the resource is not ready yet waiting for the pull request